src/badfish/main.py

```diff
@@ -2002,6 +2007,166 @@ async def delete_session(self):
             raise BadfishException(f"Failed to delete X-Auth-Token for {self.host}")
         return
 
+    async def get_scp_targets(self, op):
+        uri = "%s%s" % (self.host_uri, self.manager_resource)
+        response = await self.get_request(uri)
+        try:
+            raw = await response.text("utf-8", "ignore")
+            data = json.loads(raw.strip())
+            _ = response.status
+            filtered_data = [
+                val for key, val in data.get("Actions").get("Oem").items() if key.endswith(f"{op}SystemConfiguration")
```

Comment:
data.get("Actions").get("Oem").items()

Afraid this would throw an `AttributeError` when "Oem" is not found.

Reply:
This also applies to all the "same" comments below.